### ISSUE TITLE
Keys show type

### ISSUE THREAD

**Issue by TalanVladimir** [zmkfirmware/zmk-studio#69]
I wish You add a description of every key, like this:

![image](https://github.com/user-attachments/assets/48aed811-cbe0-4a33-92be-15960f3fe0d6)

The problem is:

![image](https://github.com/user-attachments/assets/aa2b2f7d-caaa-4ab2-acaf-aede5a08af9a)

Solution:

Remove hidden titles ;)

**Comment by averrin** [comment]
Yes, it's a good feature. For now the visual part is a bit... too minimal.

**Comment by joeynguyen** [comment]
Doesn't https://github.com/zmkfirmware/zmk-studio/issues/17 already encompass this issue?  Or is this requesting something different?
